java-checks/src/main/resources/org/sonar/l10n/java/rules/java/S5803_java.html

```diff
@@ -0,0 +1,30 @@
+<p><code>@VisibleForTesting</code>&nbsp;can be used to mark methods and fields whose visibility restrictions have been relaxed more than necessary for
+the API to allow for easier unit testing.</p>
+<p>Access to such methods and fields only possible thanks to this relaxed visibility is fine for test code, but it should be avoided in production
+code.</p>
+<p>Supported framework:</p>
+<p> * Guava (com.google.common.annotations.VisibleForTestin)</p>
```

Comment:
missing `g` at the end :)
Also, I fee that it would be nice to mark the fully qualified names with `<code>` blocks